scripts/population/regions-population

```diff
@@ -0,0 +1,118 @@
+#! /bin/bash
+set -euo pipefail
+export PGPASSWORD=$POSTGRES_PASSWORD;
+
+# Creates raster files data/populations/(data|maps)/REGIONID.tif with the population
+# values of each REGIONID in the regions table and those same values
+# scaled to byte for visualisation in mapserver respectively.
+
+BIN_PATH=${BIN_PATH:-cpp}
+SCRIPT_PATH=${SCRIPT_PATH:-scripts}
+DATA_PATH=${DATA_PATH:-data}
+
+# function raster-filename {
+#   psql -d $POSTGRES_DB -U $POSTGRES_USER -h $POSTGRES_HOST -p $POSTGRES_PORT -t -A \
+#     -c "SELECT id, filename FROM population_sources WHERE country = '$1';" | cut -d \| -f 2
+# }
+
+function target_resolution {
+    raster_name=$1
+    gdalinfo $raster_name | grep "Pixel Size" | sed -E 's/.*\(([^)]*).*/\1/g' | tr ',' ' '
+}
+
+###############################################################################
+
+echo "   -> Precalculating population aggregates per region"
+# REGIONS_QUERY="SELECT id ,country FROM regions WHERE total_population IS NULL OR max_population IS NULL OR raster_pixel_area IS NULL;"
+# if [ $# -gt 1 ]; then
+#   if [ "all" == $2 ]; then
+#     REGIONS_QUERY="SELECT id, country FROM regions;"
+#   fi
+# fi
+
+PARAM_COUNTRY=$1
+PARAM_ID_SOURCE=$2
+# echo $PARAM_COUNTRY
+# echo $PARAM_ID_SOURCE
+
+# TODO: get app folder from env
+cd /app
+
+REGIONS=$(psql -d $POSTGRES_DB -U $POSTGRES_USER -h $POSTGRES_HOST -p $POSTGRES_PORT -t -A -c "SELECT id, country FROM regions WHERE country = '$PARAM_COUNTRY';")
+echo "   -> Regions to process: $(echo $REGIONS | wc -w | tr -d '[[:space:]]')"
+
+SOURCE_ROW=$(psql -d $POSTGRES_DB -U $POSTGRES_USER -h $POSTGRES_HOST -p $POSTGRES_PORT -t -A -c "SELECT id, name, tif_file FROM population_sources WHERE id = '$PARAM_ID_SOURCE';")
+RASTER_FILENAME=`echo $SOURCE_ROW | cut -d \| -f 3`
+echo -n "   -> Filename to process: $(echo $RASTER_FILENAME) ... "
+
+if [ ! -e ${DATA_PATH}/$RASTER_FILENAME ]; then
+  echo " not found."
+  exit 1
+fi
+echo " found!"
+
+mkdir -p ${DATA_PATH}/populations/data
+mkdir -p ${DATA_PATH}/populations/maps
+
+for region in $REGIONS; do
+  ID=`echo $region | cut -d \| -f 1`
+  COUNTRY=`echo $region | cut -d \| -f 2`
+  RASTER=$DATA_PATH/$RASTER_FILENAME
+  DATA_URL="https://s3.amazonaws.com/planwise/data"
+  RASTER_URL=${DATA_URL}/$RASTER_FILENAME
+
+  if [[ ! -e $RASTER ]]; then
+    echo "   -> Downloading population file for $COUNTRY"
```

Comment:
Does downloading make sense here? The script is to register a given population .tif file, and not a population of a given country.